src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PublicSurfaceTests.cs

```diff
@@ -50,13 +50,17 @@ public void PublicSurfaceTest_Connectors()
               "Microsoft.PowerFx.Connectors.OpenApiParser",
               "Microsoft.PowerFx.Connectors.PowerFxConnectorException",
               "Microsoft.PowerFx.Connectors.PowerPlatformConnectorClient",
+              "Microsoft.PowerFx.Connectors.ReferencedEntity",
               "Microsoft.PowerFx.Connectors.RuntimeConfigExtensions",
               "Microsoft.PowerFx.Connectors.RuntimeConnectorContextExtensions",
               "Microsoft.PowerFx.Connectors.SupportsConnectorErrors",
-              "Microsoft.PowerFx.Connectors.Tabular.ConnectorDataSource",
-              "Microsoft.PowerFx.Connectors.Tabular.ConnectorServiceBase",
-              "Microsoft.PowerFx.Connectors.Tabular.ConnectorTable",
-              "Microsoft.PowerFx.Connectors.Tabular.TabularService",
+              "Microsoft.PowerFx.Connectors.TabularDataSource",
+              "Microsoft.PowerFx.Connectors.TabularRecordType",
+              "Microsoft.PowerFx.Connectors.TabularService",
+              "Microsoft.PowerFx.Connectors.TabularServiceBase",
```

Comment:
why this distinction? Ie, is is meaningful to expose both base and derived?

Reply:
There is no change here (it was public), I just moved these classes in non-Tabular namespace (it wasn't used consistently & not needed) + renamed the class

I can make it internal if you want and we'll see if need it public or not later